hover/effects/icon-transitions/_icon-setup.sass

```diff
@@ -0,0 +1,18 @@
+// Mixin to position icons
+// This mixin is used in every icon-transition mixin and must be included
+// if you are using any other icon-transition files
+
+@mixin icon-setup($icon-size: 20px, $position: right, $spacing-inner: 10px, $spacing-outer: 15px)
+    position: relative
+    @if $position == right
+        padding:
+            right: $icon-size + $spacing-outer + $spacing-outer
+    @else
```

Comment:
```sass
@else if $position == left
```
so that wrong positions aren't attributed to being on the left. You can error for that case:
```sass
@if $position == right
    padding-right: $amount
@else if $position == left
    padding-left: $amount
@else
    @error "$position must be left or right, but was #{$position}"
```

You can also use `$position` to be DRY here:
```sass
@if $position != left or $position != right
    @error "$position must be left or right, but was #{$position}"

position: relative
padding-#{$position}: $amount
```